ci/aws-analytical-env/shared/resources.yml

```diff
@@ -138,7 +138,7 @@ resources:
   - name: chs-uc-lab-repo
     type: git
     source:
-      branch: Migrating-uc-lab-aws
+      branch: master
```

Comment:
Can you keep this as is, currently we are working off that branch, since the master branch is crown jobs.